sample/db/samples/product_properties.rb

```diff
@@ -43,8 +43,7 @@
     'gender' => 'Men\'s'
   }
 ]
-
-Spree::Taxon.find_by!(name: 'Men').children.find_by!(name: 'T-shirts').products.each do |product|
+Spree::Taxon.i18n.find_by!(name: 'Men').children.i18n.find_by!(name: 'T-shirts').products.each do |product|
```

Comment:
I wonder whether it's not better to include i18n by default and have an option to opt-out of it (so you don't have to write `...i18n...` each time.

Reply:
<img width="866" alt="Screenshot 2022-08-02 at 11 52 48" src="https://user-images.githubusercontent.com/72470330/182346665-01a8d06e-7742-43d4-9a4b-c787b0433bfa.png">

I will try that.

Reply:
Doesn't work for me. There are some problems with `first_or_create!` function during sample load.